### PR TITLE
Display after stats

### PR DESCRIPTION
**Pull request by Ale282313**

### CODE REVIEW

File: app/game/views.py [hunk 1]
```diff
@@ -16,26 +16,20 @@ def meatballs_game():
...
 @game.route('/after')
 @login_required
-def after():
-    return render_template('game/after_game.html')
-
-
-@game.route('/statistics')
-@login_required
 def get_statistics():
     winner = request.args.get('winner', default = "undefined")
```
Comment (simutoni) [inline app/game/views.py]: I think you should delete the spaces around the =

File: app/static/css/after_game.css [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
-.empty {
-    min-height: 20px;
+.row-table {
+    display: table;
 }
 
-.stats-wrapper {
-    margin-bottom: 40px;
+.statistics {
+
```
Comment (simutoni) [inline app/static/css/after_game.css]: please delete the empty line

File: app/static/js/game/game.js [hunk 3]
```diff
@@ -80,7 +80,7 @@ $(document).ready(function () {
     });
 
     socket.on('291', function(data) {
-        window.location = "./statistics?winner=" + data.winner
+        window.location = "./after?winner=" + data.winner
```
Comment (simutoni) [inline app/static/js/game/game.js]: maybe you can use string concatenation from javascript str1.concat(str2, str3...)

File: app/templates/game/after_game.html [hunk 4]
```diff
@@ -4,41 +4,35 @@
...
-        </div>
+        <div class="table-wrapper">
+            <div class="container">
+                <div class="username">
+                    {{ player1_username }}:
+                </div>
+                <div class="statistics">
+                    Total shots: {{ player1_totalShots }}<br>
+                    Hit shots: {{ player1_hitShots }}<br>
+                    Number of shield activations: {{ player1_shieldActivation}}<br>
```
Comment (simutoni) [inline app/templates/game/after_game.html]: try to be consistent with white spaces: {{ x }} noy {{ x}}

File: app/templates/game/after_game.html [hunk 5]
```diff
@@ -4,41 +4,35 @@
...
             </div>
+            <br>
+             <div class="container">
+                <div class="username">
+                    {{ player2_username }}:
+                </div>
+                <div class="statistics">
+                    Total shots: {{ player2_totalShots }}<br>
+                    Hit shots: {{ player2_hitShots }}<br>
+                    Number of shield activations: {{ player2_shieldActivation}}<br>
```
Comment (simutoni) [inline app/templates/game/after_game.html]: same here

File: app/game/views.py [hunk 6]
```diff
@@ -16,26 +16,20 @@ def meatballs_game():
...
-    player1_hit_shots = request.args.get('player1_hitShots', default = 0)
-    player1_shield_activation = request.args.get('player1_shieldActivation')
-    player2_username = request.args.get('player2_username', default = "undefined")
-    player2_total_shots = request.args.get('player2_totalShots', default = 0)
-    player2_hit_shots = request.args.get('player2_hitShots', default = 0)
-    player2_shield_activation = request.args.get('player2_shieldActivation', default = 0)
-
-
-    return redirect(url_for('game.after', winner=winner,
+    winner = request.args.get('winner', "undefined")
```
Comment (cristimihok) [inline app/game/views.py]: Why do you assign to new variables? You could unload the parameters directly in the `render_template` function.

File: app/static/js/game/game.js [hunk 7]
```diff
@@ -73,22 +73,23 @@ $(document).ready(function () {
...
-            +"&player1_username=" + data.player1_username
-            +"&player1_totalShots=" + data.player1_totalShots
-            +"&player1_hitShots=" + data.player1_hitShots
-            +"&player1_shieldActivation=" + data.player1_shieldActivation
-            +"&player2_username=" + data.player2_username
-            +"&player2_totalShots=" + data.player2_totalShots
-            +"&player2_hitShots=" + data.player2_hitShots
-            +"&player2_shieldActivation=" + data.player2_shieldActivation;
+        window.location = "";
+        window.location.concat("./after?winner=", data.winner,
```
Comment (cristimihok) [inline app/static/js/game/game.js]: This is nice. Now that you know how to do it in plain js now please check JQuery's `param` method. `$.param(object)`

**Pull request merged**